src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -568,7 +571,10 @@ describe('XhrPlugin tests', () => {
         plugin.disable();
 
         // Assert
-        expect(record).not.toHaveBeenCalled();
```

Comment:
question: if the session is not being recorded, shouldn't `record` not be invoked at all?

Reply:
I changed the logic to check for session recording inside Xhr before reading this comment. I can change it back :/